edb/server/protocol/binary.pyx

```diff
@@ -1194,6 +1194,8 @@ cdef class EdgeConnection:
         return query_unit
 
     def signal_side_effects(self, side_effects):
+        if self.server.is_shutting_down():
```

Comment:
This would be `false` in bootstrap mode.  I think we need a single condition, e.g. `is_active()` to cover both the "is_shutting_down()" and "not started at all" case.